modules/vsphere-enabling-multiple-layer2-networks.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+// * post_installation_configuration/vsphere-failure-domain-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="vsphere-enabling-multiple-layer2-network_{context}"]
+= Enabling a multiple layer 2 network for your cluster
+
+You can configure your cluster to use a multiple layer 2 network configuration, so that data transfer among nodes can span across multiple networks. 
+
+.Prerequisites
+* You configured network connectivity among machines so that cluster components can communication with each other.
+
+.Procedure
+* If you installed your cluster with the installer-provisioned infrastructure method, complete the following actions to meet your needs:
+** Place the API VIP, the Ingress VIP, and their targets in the same layer 2 network. This configuration causes the following components to share the same layer 2 network: (1) the API VIP and control plane nodes, and (2) the Ingress VIP and any nodes that host the Ingress router pods.
+** If you need to move the Ingress VIP and its router pods to specific nodes, you must create infrastructure machine sets that host these infrastructure components in the target nodes.
+** If you need provision workloads to additional layer 2 networks, you can create compute machine sets on vSphere than move these workloads to your target layer 2 networks. 
```

Comment:
```suggestion
* If you installed your cluster with the installer-provisioned infrastructure method, you must ensure that all control plane nodes share a common layer 2 network. In addition, ensure compute nodes where ingress pods may be scheduled share a common layer 2 network.  
** If you need compute nodes to span multiple layer 2 networks infrastructure nodes can be created to host ingress pods.  
** If you need to provision workloads across additional layer 2 networks, you can create compute machine sets on vSphere than move these workloads to your target layer 2 networks. 

```

Reply:
just an idea after reading through it

Reply:
Hi @rvanderp3 

Thanks. I implemented your suggestions. Please let me know your thoughts.